src/main.rs

```diff
@@ -12,26 +13,118 @@ const VERSION: &str = env!("TYPSTFMT_VERSION");
 const CONFIG_PATH: &str = "typstfmt-config.toml";
 const HELP: &str = r#"Format Typst code
 
-usage: typstfmt [options] <file>...
+usage: typstfmt [options] [file]...
```

Comment:
I usually see docs with <input> rather than [input], are you sure this is the way?

Reply:
`<input>` would be for a required arument, `[input]` for an optional.

One fixup might be to move the dots inside: `[file...]`

Reply:
Consider it done!